pkgs/development/libraries/glibc/common.nix

```diff
@@ -165,10 +165,18 @@ stdenv.mkDerivation ({
     "OBJCOPY=${stdenv.cc.targetPrefix}objcopy"
   ];
 
+  postInstall = (args.postInstall or "") + ''
+    moveToOutput bin/getent $getent
```

Comment:
Is there any downside (or even upside...) to symlinking `getent` back to the bin output?

Reply:
An upside would be that we're less likely to break packages that use getent; a downside would be that if such packages exist, they should probably be using the `getent` package from toplevel since getting it from glibc is incorrect on multiple platforms.

Reply:
As stated in the release-notes by Linus, using `getent` rather than `getBin glibc` is actually the right way to reference `getent`, so that's a good thing -> resolving this comment.